.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: v5.0.0
     hooks:
     - id: trailing-whitespace
-    - id: end-of-file-fixer
+    # - id: end-of-file-fixer
```

Comment:
Note to self: Needs some further thought to fix for windows users... idk really what's going on...